app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/firestore/FirestoreDatabaseService.kt

```diff
@@ -309,6 +311,22 @@ class FirestoreDatabaseService internal constructor(
         return documentRef.get().await().toObject(UserSchema::class.java)!!.toPublic()
     }
 
+    //TODO Optimize transaction to db
```

Comment:
What about this?

Reply:
Yes It's adressed I only update the admin Instead of updating all the user variables